docs/setup/storage/storage.md

```diff
@@ -53,6 +54,20 @@ To enable versioning for all your local notebooks though a standard Git reposito
 
 </br>
 
+## Notebook Storage in Hdfs repository <a name="Hdfs"></a>
+
+Notes may be stored in hdfs, so that multiple zeppelin instance can share the same notes. It supported all the versions of hadoop 2.x. If you use `HdfsNotebookRepo`, then `zeppelin.notebook.dir` is the path on hdfs. And you need to specify `HADOOP_CONF_DIR` in `zeppelin-env.sh` so that zeppelin can find the right hadoop configuration files.
```

Comment:
capital `Zeppelin`

`Notes may be stored in hdfs, so that multiple zeppelin instance can share the same notes.`
- if this is the intended goal, will there be any race condition on write when several instances are changing it the same time?